src/app/visual/sectioned-form/sectioned-form-visual.component.ts

```diff
@@ -7,4 +8,10 @@ import { ChangeDetectionStrategy, Component} from '@angular/core';
 })
 export class SectionedFormVisualComponent {
   public maintainSectionContent: boolean = false;
+
+  constructor(@Optional() private themeSvc: SkyThemeService) {}
+
+  public themeSettingsChange(themeSettings: SkyThemeSettings): void {
+    this.themeSvc.setTheme(themeSettings);
```

Comment:
Since theme service is optional, this should check if `this.themeSvc` is set.

Reply:
Yup - good call - fixed.